.github/workflows/linux_android_arm64.yml

```diff
@@ -0,0 +1,178 @@
+name: Linux Android ARM64 (Ubuntu 20.04, Python 3.11)
+on:
+  schedule:
+    # run daily at 00:00
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+#  pull_request:
+#    paths-ignore:
+#      - '**/docs/**'
+#      - 'docs/**'
+#      - '**/**.md'
+#      - '**.md'
+#      - '**/layer_tests_summary/**'
+#      - '**/conformance/**'
+#  push:
+#    paths-ignore:
+#      - '**/docs/**'
+#      - 'docs/**'
+#      - '**/**.md'
+#      - '**.md'
+#      - '**/layer_tests_summary/**'
+#      - '**/conformance/**'
+#    branches:
+#      - master
+
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}-linux-android-arm64
+  cancel-in-progress: true
+
+jobs:
+  Build:
+    # TODO: remove. Temporary measure to prevent the workflow from scheduling on forks.
+    if: ${{ github.repository_owner == 'openvinotoolkit' }}
+    defaults:
+      run:
+        shell: bash
+    runs-on: ubuntu-20.04-8-cores
+    env:
+      CMAKE_BUILD_TYPE: 'Release'
+      CMAKE_GENERATOR: 'Ninja'
+      CMAKE_CXX_COMPILER_LAUNCHER: ccache
+      CMAKE_C_COMPILER_LAUNCHER: ccache
+      BUILD_TYPE: Debug
+      OPENVINO_REPO: ${{ github.workspace }}/openvino
+      VCPKG_ROOT: ${{ github.workspace }}/vcpkg
+      BUILD_DIR: ${{ github.workspace }}/build
+      INSTALL_DIR: ${{ github.workspace }}/install
+      OV_TEMP: ${{ github.workspace }}/openvino_temp
+      ANDROID_TOOLS: ${{ github.workspace }}/android_tools
+      ANDROID_NDK_HOME: ${{ github.workspace }}/android_tools/ndk-bundle
+      ANDROID_SDK_VERSION: 29
+      ANDROID_ABI_CONFIG: arm64-v8a
+    steps:
+      - name: Clone OpenVINO
+        uses: actions/checkout@v3
+        with:
+          path: 'openvino'
+          submodules: 'recursive'
+
+      - name: Clone VCPKG
+        uses: actions/checkout@v3
+        with:
+          repository: 'microsoft/vcpkg'
+          path: 'vcpkg'
+          fetch-depth: '0'
+          submodules: 'recursive'
+
+      - name: Create Directories
+        run: |
+          mkdir -p ${{ env.BUILD_DIR }}
+          mkdir -p ${{ env.INSTALL_DIR }}
```

Comment:
can we eliminate this step ? (I hope the directories are automatically created) or at least merge with other steps.

Ideally, we need steps which perform some interesting steps of pipeline (e.g. clone, install dependencies, build, test, package) and avoid dummy steps like create dir, print some values or simiar